cms/tests/test_page.py

```diff
@@ -501,6 +501,16 @@ def test_publish_page_regression_6188(self):
         # Now there should be "en", "de" and "fr" on the public page
         self.assertSequenceEqual(sorted(page.publisher_public.get_languages()), ['de', 'en', 'fr'])
 
+    def test_create_page_with_position_regression_6345(self):
+        # ref: https://github.com/divio/django-cms/issues/6345
+        parent = create_page("p", "nav_playground.html", "en")
+        rightmost = create_page("r", "nav_playground.html", "en",parent=parent)
```

Comment:
please add space here

Reply:
please also move this test to test_api https://github.com/divio/django-cms/blob/develop/cms/tests/test_api.py